packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -76,6 +76,12 @@ export const PaymentUpdateForm = ({
     .unix(customerSubscription.current_period_end)
     .format('MM/DD/YYYY');
 
+  // https://github.com/iamkun/dayjs/issues/639
+  const expirationDate = dayjs()
+    .set("month", Number(exp_month))
+    .set("year", Number(exp_year))
+    .format("MMMM YYYY");
```

Comment:
Just read this, are there locales where this ordering is swapped? I know we don't have to worry about it for MVP, something to keep in mind when expanding into other countries.